src/main/java/com/gluonhq/substrate/target/MacOSTargetConfiguration.java

```diff
@@ -143,13 +143,30 @@ String getAdditionalSourceFileLocation() {
 
     @Override
     public boolean packageApp() throws IOException, InterruptedException {
-        createAppBundle();
+        boolean sign = !projectConfiguration.getReleaseConfiguration().isSkipSigning();
+        createAppBundle(sign);
+
+        String packageTypes = projectConfiguration.getReleaseConfiguration().getPackageTypes();
+        if (packageTypes == null || packageTypes.isEmpty()) {
+            // if it is not set, do nothing
+            return true;
+        } else if (!(packageTypes.contains("pkg") || packageTypes.contains("dmg"))) {
+            // if it is set but doesn't ask for pkg or dmg, do nothing
+            return true;
+        }
 
+        boolean result = true;
         Packager packager = new Packager(paths, projectConfiguration);
-        return packager.createPackage(!projectConfiguration.getReleaseConfiguration().isSkipSigning());
+        if (packageTypes.contains("pkg")) {
+            result = packager.createPackage(sign);
+        }
+        if (result && packageTypes.contains("dmg")) {
```

Comment:
this is the reason I would have preferred to allow only one type: pkg (either standalone or appstore) or dmg

Reply:
I'll refactor it